### PR TITLE
Update this plugin to the v2.0 of the plugin api

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: spec/filters/useragent_spec.rb [hunk 1]
```diff
@@ -120,10 +120,10 @@
 
     sample "Mozilla/5.0 (X11; Linux i686) AppleWebKit/537.31 (KHTML, like Gecko) Chrome/26.0.1410.63 Safari/537.31" do
       insist { subject.to_hash }.include?("message")
-      insist { subject["message"]["name"] } == "Chrome"
-      insist { subject["message"]["os"] } == "Linux"
-      insist { subject["message"]["major"] } == "26"
-      insist { subject["message"]["minor"] } == "0"
+      insist { subject.get("message")["name"] } == "Chrome"
```
Comment (talevy) [inline spec/filters/useragent_spec.rb]: from our discussion about what is "proper" usage of the event api, shouldn't this change into

```
subject.get("[message][name]")
```

what do you think?

**Comment by ph** [comment]
updated with your changes.

**Comment by ph** [comment]
@talevy ^

File: spec/filters/useragent_spec.rb [hunk 2]
```diff
@@ -17,10 +17,10 @@
 
     sample "Mozilla/5.0 (X11; Linux i686) AppleWebKit/537.31 (KHTML, like Gecko) Chrome/26.0.1410.63 Safari/537.31" do
       insist { subject }.include?("ua")
-      insist { subject["ua"]["name"] } == "Chrome"
-      insist { subject["ua"]["os"] } == "Linux"
-      insist { subject["ua"]["major"] } == "26"
-      insist { subject["ua"]["minor"] } == "0"
+      insist { subject.get("ua")["name"] } == "Chrome"
```
Comment (talevy) [inline spec/filters/useragent_spec.rb]: changes still needed here? and below

Comment (ph) [inline spec/filters/useragent_spec.rb]: fixed, I just change at the end of the file

**Comment by talevy** [comment]
LGTM

**Pull request merged**